sdk-api-src/content/traceloggingprovider/nf-traceloggingprovider-tracelogging_define_provider.md

```diff
@@ -64,8 +64,20 @@ The name of the TraceLogging provider. This must be a string literal--do not use
 
 ### -param providerId [in]
 
-The GUID for the provider.
+The GUID for the provider. This value is the result of applying the RFC 4122 algorithm to the provider's name. 
```

Comment:
This is somewhat oversimplified.
- This can be any GUID, e.g. a GUID generated using guidgen SDK tool or uuidgen.org.
- However, Microsoft has established a convention for generating the GUID using a hash of the provider name, which enables several benefits (easier to remember just the name, works with tools like tracelog, traceview, EventSource, WPR).
- The GUID generation algorithm is almost (but not quite) compatible with RFC 4122 v5 UUID.